Library/Homebrew/test/diagnostic_checks_spec.rb

```diff
@@ -24,6 +24,7 @@
   end
 
   specify "#check_access_directories" do
+    skip "User is root." if Process.euid == 0
```

Comment:
`User is root so everything is readable`.

Reply:
Fixed.